README.md

```diff
@@ -75,8 +75,8 @@ Make sure you have a supported ruby installed, cd into your cucumber-rails repos
 
     gem install bundler
     bundle install
-    bin/install_geckodriver.sh
-    bin/install_webpacker.sh
+    ./bin/install_geckodriver.sh
+    ./bin/install_webpacker.sh
```

Comment:
This `./` shouldn't be needed, right? Is there a circumstance where the old instructions are a problem?

Reply:
It was for me, but I'm on ubuntu. It's my own fault really. I couldn't get yarn to update or rails6 to work, and then I manually fixed everything. When I re-checked this file I realised I hadn't ran the commands properly.

I can remove them if you'd prefer. Just helps me on Bionic run them effectively as they're shells.